lang/src/org/partiql/lang/eval/Exceptions.kt

```diff
@@ -39,7 +39,7 @@ open class EvaluationException(message: String,
     constructor(cause: Throwable,
```

Comment:
Aren't we removing the constructor on line 30 as well?

Reply:
Sure. Thanks, David. I will update the PR.